configCivcraft.yml

```diff
@@ -4640,23 +4637,23 @@ recipes:
         amount: 512
     factory: Flower Gardening
   Upgrade_to_Gem_Extractor:
-    production_time: 4h
+    production_time: 8h
     type: UPGRADE
     name: Upgrade to Gem Extractor
-    fuel_consumption_intervall: 2m
+    fuel_consumption_intervall: 4m
     input:
       diamond:
         material: DIAMOND_BLOCK
-        amount: 64
+        amount: 16
```

Comment:
That would be the case if there were no previous factories. but good point 16 is too low anyway.